draft-ietf-core-groupcomm-bis.md

```diff
@@ -1115,15 +1135,278 @@ Service discovery is the discovery and identification of particular services hos
+ |          |          |          |          |          |          |
+~~~~~~~~~~~
+{: #fig-lights-response title="Lights (Optionally) Respond to Multicast CoAP Request" artwork-align="center"}
+
+Another, but similar, lighting control use case is shown in {{fig-light-switch-control-msg-2}}. In this case, a controller connected to the network backbone sends a CoAP group communication request to turn on all lights in Room-A. Every light sends back a CoAP response to the controller after being turned on.
+
+~~~~~~~~~~~ aasvg
+                                                            CoAP
+                                                   Network  Controller
+Light-1   Light-2    Light-3     Rtr-1      Rtr-2  Backbone Client
+ |          |          |           |          |          |        |
+ |          |          |           |          |    COAP NON Mcast(PUT,
+ |          |          |           |          |    Payload=lights ON)
+ |          |          |           |          |          |<-------|
+ |          |          |           |<----------<---------|        |
+ |<--------------------------------|          |          |        |
+ ON         |          |           |          |          |        |
+ |          |<----------<---------------------|          |        |
+ |          ON         ON          |          |          |        |
+ ^          ^          ^           |          |          |        |
+ .......................           |          |          |        |
+ :   Lights in Room-A  :           |          |          |        |
+ :   turn on (nearly   :           |          |          |        |
+ :   simultaneously)   :           |          |          |        |
+ :.....................:           |          |          |        |
+                                   |          |          |        |
+ |          |          |           |          |          |        |
+ |          |          |           |          |          |        |
+ |    COAP NON (2.04 Changed)      |          |          |        |
+ |-------------------------------->|          |          |        |
+ |          |          |           |-------------------->|        |
+ |          |  COAP NON (2.04 Changed)        |          |------->|
+ |          |-------------------------------->|          |        |
+ |          |          |           |          |--------->|        |
+ |          |          | COAP NON (2.04 Changed)         |------->|
+ |          |          |--------------------->|          |        |
+ |          |          |           |          |--------->|        |
+ |          |          |           |          |          |------->|
+ |          |          |           |          |          |        |
+~~~~~~~~~~~
+{: #fig-light-switch-control-msg-2 title="Controller on Backbone Sends Multicast Control Message" artwork-align="center"}
+
+#### Lighting Control in MLD-Enabled Network # {#example-lighting-control-mld}
```

Comment:
I see some issues with this example. It would be best to just remove it, I think:
* as stated, 6LoWPAN nodes today typically don't implement MLDv2 so it would be confusing to have an example here where they do.
   *  Today's 6lowpan nodes have other mechanisms to subscribe to multicast addresses. E.g. Thread has a custom mechanism; and RPL defined a method for it, and there is recent work also in 6lo that we don't want to interfere with.
* MLDv2 is only defined to operate on a single link (LL). So it's use in a mesh network structure is questionable.
* It's not correct protocol to use 2 layers of MLDv2 as shown here. Normally a host would use MLDv2 to talk to a router and the router would use PIM (or other equivalent) to talk to other routers.

Reply:
Agree! https://github.com/core-wg/groupcomm-bis/commit/a80308b849ab9f6e0cd16444f93cff01c7c6651d  has removed that appendix and made some early rephrasing to introduce the remaining text in a consistent way.